projects/Mallard/src/hooks/use-issue-summary.tsx

```diff
@@ -58,20 +70,29 @@ const IssueSummaryProvider = ({ children }: { children: React.ReactNode }) => {
     }
 
     useEffect(() => {
-        NetInfo.addEventListener(({ isConnected }) => {
-            if (!hasConnected.current) {
-                hasConnected.current = isConnected
+        if (data) {
```

Comment:
Im a bit lost here in this file without testing it. Do we still get the latest editions when we go from offline to online and when the app stage changes back to active?